spec/policies/organisation_policy_spec.rb

```diff
@@ -86,8 +86,13 @@
     context 'for super org admins' do
       let(:user) { create(:super_org_admin, organisation: org_root_one) }
 
-      it 'is empty' do
-        expect(resolved_scope).to be_empty
+      it 'includes own org and child orgs' do
```

Comment:
I don't know if this is true.  The `index?` method for OrganisationPolicy suggests that only super admins and admins can see a list of organisations, so I think the scope should match that and return nothing.  Or! the `index?` method  should return true for super org admins to match the change you've made here.

I suspect however, changing the `index?` to return `true` will have more far reaching repercussions than changing the scope to return `[]`, as I suspect it's used to control what controller and view actions you're allowed to see, and I don't think we want to let super org admins see a list of orgs (but then again, maybe we do - they can't edit those orgs, but it might be useful for them to see the list of orgs they're able to manage users for).

Reply:
I've made the scope match the `index?` method (returns `scope.none`)